cc/main.cc

```diff
@@ -331,6 +348,13 @@ class SelfPlayer {
     const bool use_ansi_colors = isatty(fileno(stderr));
 
     GameOptions game_options;
+    int row_cursor = 0;
+    std::vector<std::string> bigtable_spec = absl::StrSplit(FLAGS_output_bigtable, ',');
+    bool use_bigtable = bigtable_spec.size() == 3;
+    if (!bigtable_spec.empty() && !use_bigtable) {
+      std::cerr << "Bigtable output must be project,instance,table" << std::endl;
```

Comment:
Let's make this a fatal error: `MG_FATAL() << ...`

Reply:
Good idea. Will return early upon this error as well. Since the Bigtable write was optional and in parallel, I had let it go by with a warning, but I agree that if the specification is given and is wrong, it's better to stop.